modules/ROOT/pages/sap/sap-connector-examples-5.0

```diff
@@ -0,0 +1,233 @@
+= SAP Connector Examples - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Establish a SAP Connection
+
+Create a Mule app that establishes a connection to SAP and receives a function using the SAP connector's *Function source* operation. The *Function source* operation passes the export parameter back to SAP. 
+
+. Create a new Mule project in Anypoint Studio.
```

Comment:
(I think it's good to locate the user, esp. in Step 1, before giving them a chore:
```suggestion
. In Anypoint Studio, create a new Mule project.
```